docs/en/ingest-management/data-streams.asciidoc

```diff
@@ -298,6 +298,8 @@ This tutorial explains how to apply a custom {ilm-init} policy to an integration
 **Goal:** Customize the {ilm-init} policy for the `system.network` data stream in the `production` namespace.
 Specifically, apply the built-in `90-days-default` {ilm-init} policy so that data is deleted after 90 days.
 
+NOTE: If you cloned an index template to customize the data retention policy on an {es} version prior to 8.13, you must update the index template in the clone to use the `ecs@mappings` component template on {es} version 8.13 or later. See "Update index template cloned before {es} 8.13" section for the step-by-step instructions.
```

Comment:
```suggestion
NOTE: If you cloned an index template to customize the data retention policy on an {es} version prior to 8.13, you must update the index template in the clone to use the `ecs@mappings` component template on {es} version 8.13 or later. See <<data-streams-pipeline-update-cloned-template-before-8.13,Update index template cloned before {es} 8.13>> for the step-by-step instructions.
```
This will add a link to the new section.